package.json

```diff
@@ -12,6 +12,7 @@
     "url": "https://github.com/Supergiovane/KNXUltimate"
   },
   "scripts": {
+    "buildall": "npm run clean && npm run build && npm run buildexamples",
     "build": "tsc -p tsconfig.build.json",
     "buildexamples": "tsc -p tsconfig.examples.json",
```

Comment:
`node -r esbuild-register examples/sample.ts`

Reply:
Please remove the `buildall` and `buildexamples` scripts and also `tsconfig.examples.json`

Reply:
I must compile it for the changes to become active.

Reply:
@Supergiovane That's not true, you don't need to compile it you can run it without compiling.

Compile ts to js is only needed when you need to deploy the application to NPM

Reply:
Boh, the only way i can see the changes i made, is to recompile.
Even if i insert a console.log("banana") into an example's code, i can see banana only after recompiling.
Probably, i have some misconfiguration in my vscode json.

Reply:
Can i resolve this conversation ?

Reply:
Also if you want to debug this insed of adding console.log you can just add the flag `--inspect` to the previous command and it will start with debugger attached and you can put breakpoints on your code in vscode to stop execution or better add logpoint that works like logs but only on debug side

Reply:
Hi
i just press F5.
Using node `-r esbuild-register examples/gatewaydescription.ts` works even without compiling.
The rest of buildall command is useful, because it first removes the build. Maybe i can add the `npm run clean` directly in the "build" script and delete the buildall.
It's OK?

Reply:
Yes you can delete the buildall and buildexamples scripts along with tsconfig.examples.json and add clean before build action